test/e2e/backup_test.go

```diff
@@ -148,10 +148,21 @@ func testEtcdBackupOperatorForS3Backup(t *testing.T) string {
 // testEtcdRestoreOperatorForS3Source tests if the restore-operator can restore an etcd cluster from an S3 restore source
 func testEtcdRestoreOperatorForS3Source(t *testing.T, s3Path string) {
 	f := framework.Global
+	suffix := fmt.Sprintf("-%d", rand.Uint64())
+	clusterName := "restore-tls-test" + suffix
+	memberPeerTLSSecret := "restore-etcd-peer-tls" + suffix
+	memberClientTLSSecret := "restore-etcd-server-tls" + suffix
+	operatorClientTLSSecret := "restore-etcd-client-tls" + suffix
+	err := e2eutil.PrepareTLS(clusterName, f.Namespace, memberPeerTLSSecret, memberClientTLSSecret, operatorClientTLSSecret)
+	if err != nil {
+		t.Fatal(err)
+	}
 
 	restoreSource := api.RestoreSource{S3: e2eutil.NewS3RestoreSource(s3Path, os.Getenv("TEST_AWS_SECRET"))}
-	er := e2eutil.NewEtcdRestore("test-etcd-restore-", "3.2.11", 3, restoreSource)
-	er, err := f.CRClient.EtcdV1beta2().EtcdRestores(f.Namespace).Create(er)
+	er := e2eutil.NewEtcdRestore("", "3.2.11", 3, restoreSource)
```

Comment:
Why adds version in the parameter?
If unnecessary, can we have this fixed in this PR?

Reply:
prefer another pr since it is not related this tls pr.

Reply:
Sure.